apps/app/src/app/pages/pages.component.ts

```diff
@@ -28,7 +28,9 @@ export class PagesComponent {
     private router: Router,
     private languageService: LanguageService,
     public settingsService: SettingsService
-  ) {
+  ) {}
+
+  ngOnInit(): void {
     setInterval(() => {
```

Comment:
Je ne pense pas que ce soit nécessaire de le faire chaque seconde, surtout si tu ne les affiches pas. Autant le mettre toutes les minutes et dans le pire des cas tu aura 59sec de retard sur l'heure officielle

Reply:
Elle est affichée dans la navbar et le temps donné est en milliseconde donc 1000 c'est une seconde * 60 on est sur la minute, je me trompe ?

Reply:
Ah oui effectivement, j'ai mis ce commentaire car j'ai vu le nom de ton test "should update date every second" donc j'ai cru que tu affichais la date toutes les secondes !